package.json

```diff
@@ -20,9 +21,6 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "devDependencies": {
-    "react-scripts": "latest"
```

Comment:
why did we remove latest react-scripts?

Reply:
i didnt but it doesnt matter

Reply:
suggest commit for that also if u want

Reply:
@nithin554 suggest commit for this plz if required

Reply:
run with latest and test code first

Reply:
sometimes we might have compatibility issues

Reply:
ok i ll try in code space, I have to npm i before running?

Reply:
@nithin554